src/loader/gltf-loader.ts

```diff
@@ -34,14 +34,14 @@ class glTFExternalResourceLoader implements glTFResourceLoader {
     if (!this._loader.resources[url]) {
       // The resource does not exists and needs to be loaded.
       // @ts-ignore
-      this._loader.add({ parentResource: this._resource, url, onComplete })
+      this._loader.add({ parentResource: this._resource, name: url, url: url, onComplete: onComplete })
     } else if (this._loader.resources[url].data) {
       // The resource already exists, just use that one.
       onComplete(this._loader.resources[url])
     } else {
       // The resource is in queue to be loaded, wait for it.
       let binding = this._loader.onProgress.add((_, resource) => {
-        if (resource.url === url) {
+        if (resource.name === url) {
```

Comment:
Double-checking that this condition is really correct?

Reply:
Good point, it does depend how it got in the queue and whether the expected name convention was assigned at that time (e.g. line 37).  An OR may be more appropriate, as either are likely acceptable outcomes.

if ( resource.url == url || resource.name == url )